.gitignore

```diff
@@ -4,3 +4,4 @@
 coverage/*
 notes.md
 tests/fixtures/cassette*
+composer.lock
```

Comment:
https://getcomposer.org/doc/02-libraries.md#lock-file - i would go further than that and say its harmful as it hides possible problems with the things specified in composer.json. a project using this library will never use your lock file so it could end up with different versions than the ones from the lock file.